src/main/resources/db/migration/V2_6__add_column_veileder_ident.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE MELDING
+ADD COLUMN veileder_ident VARCHAR(7) DEFAULT '';
```

Comment:
Usikker på om man skulle sette `DEFAULT NULL` eller `DEFAULT ''`. Hvis `NULL` må vel typen være `String?` vil jeg tro? Ville unngå det, egentlig.

Reply:
Er det bedre å sjekke om en string er tom enn å sjekke om den er null? 🤔

Reply:
Jeg tenker at den kan være DEFAULT NULL. Blir det ikke NULL (for meldinger fra behandler) eller en veileder-ident (for meldinger til behandler) som lovlige verdier her?

Reply:
Ja, "ADD COLUMN veileder_ident VARCHAR(7)" lager en kolonne som tillater null-verdier (hvis man ikke vil ha dette må man eksplisitt si "ADD COLUMN veileder_ident VARCHAR(7) NOT NULL".

Reply:
Ja, jeg kan endre til at det er `null` i stedet, og også `String?`